maquetado/index.html

```diff
@@ -49,25 +53,25 @@ <h2 class="title">Back-End Tools</h2>
   <div class="container">
     <div class="wrap">
       <div class="technology">
-        <img class="logo" src="./assets/rails_icon.png">
+        <img class="logo" src="./assets/rails_icon.png" alt="technology logo">
         <h4 class="technology-title">Ruby on Rails</h4>
         <h4 class="technology-release">Release:2005</h4>
         <p class="technology-description">Ruby on Rails, or Rails, is a server-side web application framework written in Ruby. Rails is an MVC framework, providing default structures for a database, a web service, and web pages.</p>
       </div>
       <div class="technology">
-        <img class="logo" src="./assets/express_icon.svg">
+        <img class="logo" src="./assets/express_icon.svg" alt="technology logo">
         <h4 class="technology-title">NodeJs</h4>
         <h4 class="technology-release">Release:2009</h4>
         <p class="technology-description">Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Uses an event-driven, non-blocking I/O model taht makes it lightweight and efficient.</p>
       </div>
       <div class="technology">
-        <img class="logo" src="./assets/aspnet_icon.png">
+        <img class="logo" src="./assets/aspnet_icon.png" alt="technology logo">
         <h4 class="technology-title">ASP.NET Core</h4>
         <h4 class="technology-release">Release:2016</h4>
         <p class="technology-description">ASP.NET Core is the next generation of ASP.NET. It is a modular framework that runs on both the full .NET Framework, on Windows, and the cross-platform .NET Core.</p>
       </div>
       <div class="technology">
-        <img class="logo" src="./assets/django_icon.svg">
+        <img class="logo" src="./assets/django_icon.svg" alt="technology logo">
```

Comment:
Same as the content layout PR. Alt attribute is used to give some context about what the image is about, but let's do this fix in content layout PR.